editor.planx.uk/src/@planx/components/MultipleFileUpload/Modal.tsx

```diff
@@ -163,6 +163,7 @@ const SelectMultiple = (props: SelectMultipleProps) => {
         What does this file show?
       </InputLabel>
       <Select
+        native={false}
```

Comment:
This is a bit of a silly change (it's just explicitly defining the default prop value) but one i think we're going to need to address in the future and this feels like a helpful way to remember.

This prop was brought to my attention when inspecting the DOM for testing (there wasn't actually a `<select>`!). While I ended up adding `data-testid` to `inputProps` to get test selectors working, I think we'll eventually want this prop set to `true` for accessibility/screen reading devices. See this thread: https://stackoverflow.com/a/55185837

Whenever we come back to this (wait to see if flagged during a11y audit?), there were a couple immediate things I noticed when locally switching to `native={true}`: console throws error that our `options` are invalid because they include list subheaders, the "what does this file show" label/placeholder becomes disconnected, and styles overall get really wonky.

Reply:
Thanks for the investigation and explanation 👌 

Happy with the "wait and see" approach here as we know that an audit will be around shortly.